src/libraries/Directory.Build.targets

```diff
@@ -13,6 +16,8 @@
   <Import Project="..\..\Directory.Build.targets" />
 
   <PropertyGroup>
+    <!-- TODO-NULLABLE: Remove once netstandard has annotations for Debug.Assert, string.IsNullOrEmpty, etc. -->
```

Comment:
I think we can remove this comment as I don't thin `NS` will ever have this annotations. cc: @stephentoub ?